_data/simple-icons.json

```diff
@@ -9274,6 +9274,11 @@
             "source": "https://www.make.com/en/press",
             "guidelines": "https://www.make.com/en/press"
         },
+        {
+            "title": "Makedeb",
+            "hex": "000000",
```

Comment:
This definitely doesn't seem like the right brand color, please read through https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines. Not sure what source you used (see my other comment), but if it was one with a gradient (e.g. https://github.com/makedeb/brand/blob/main/assets/transparent.svg) we have instructions on selecting the gradient midpoint in the color guidelines section I linked above.

Reply:
Fixed.
https://github.com/simple-icons/simple-icons/pull/11323/commits/c10eae941ec193ec37c85f1930e43dca7f5b7517

Reply:
New preview.
![image](https://github.com/user-attachments/assets/7e9c6a40-a09b-474f-9cae-14e24b97298b)